app/views/course/condition/achievements/_achievement.html.slim

```diff
@@ -0,0 +1,13 @@
+div
+  label Achievement
+  '
+  = achievement.achievement.title
+
+  br
```

Comment:
Follow the normal design for all the pages. I don't think we `br` after a title.